src/translate/translate.py

```diff
@@ -614,7 +615,8 @@ def pddl_to_sas(task):
         with timers.timing("Reordering and filtering variables", block=True):
             variable_order.find_and_apply_variable_order(
                 sas_task, options.reorder_variables,
-                options.filter_unimportant_vars)
+                options.filter_unimportant_vars,
```

Comment:
I am aware this existing line is inconsistent with the recommendations I'm making here, but it's the exception rather than the rule in the current code. (I grepped for all uses of `options`.)